aviary/variable_info/variable_meta_data.py

```diff
@@ -203,6 +203,122 @@
     default_value=1.0,
 )
 
+#  ____            _     _
+# |  _ \          | |   | |
+# | |_) |   __ _  | |_  | |_    ___   _ __   _   _
+# |  _ <   / _` | | __| | __|  / _ \ | '__| | | | |
+# | |_) | | (_| | | |_  | |_  |  __/ | |    | |_| |
+# |____/   \__,_|  \__|  \__|  \___| |_|     \__, |
+#                                             __/ |
+#                                            |___/
+# =================================================
+add_meta_data(
+    Aircraft.Battery.ADDITIONAL_MASS,
+    meta_data=_MetaData,
+    historical_name={"GASP": None,
+                     "FLOPS": None,
+                     "LEAPS1": 'aircraft.inputs.L0_battery.weight_offset'
+                     },
+    units='lbm',
+    desc='mass of non energy-storing parts of the battery',
+    default_value=0.0,
+)
+
+add_meta_data(
+    Aircraft.Battery.DISCHARGE_LIMIT,
+    meta_data=_MetaData,
+    historical_name={"GASP": None,
+                     "FLOPS": None,
+                     "LEAPS1": 'aircraft.inputs.L0_battery.depth_of_discharge'
+                     },
+    units='unitless',
+    desc='default constraint on how far the battery can discharge, as a percentage of '
+         'total energy capacity'
```

Comment:
a default of 0 or 20% would be reasonable

Reply:
Added 20%!